addons/website/static/src/scss/website.scss

```diff
@@ -1066,13 +1066,26 @@ header {
 } @else if o-website-value('header-template') == 'sidebar' {
     @include media-breakpoint-up(lg) {
         #wrapwrap {
-            // Hack: padding is used by layout option (boxed, etc) so use
-            // border here to be able to combine the effect.
+            // Hack: padding is also used by layout option (boxed, etc), need to
+            // combine both options/effects...
             @if $-hamburger-right {
                 border-right: o-website-value('sidebar-width') solid transparent;
             } @else {
                 border-left: o-website-value('sidebar-width') solid transparent;
             }
+            // ... but don't use it on Safari, it doesn't work, the menu is
+            // somehow broken and not visible.
+            &.o_safari_browser {
+                @if o-website-value('layout') != 'full' {
```

Comment:
Despite what your commit explains, the bug is also there when the layout is full and this commit seems only treating the case of sidebar + boxed (which probably no one ever want) if I am not mistaken? :grimacing: I guess you just forgot the `else` part here (where you want to use padding instead of border and not even need `calc`)? :+1: